src/Types/Transaction.purs

```diff
@@ -775,7 +775,7 @@ instance Show TransactionMetadatumLabel where
 data TransactionMetadatum
   = MetadataMap (Map TransactionMetadatum TransactionMetadatum)
   | MetadataList (Array TransactionMetadatum)
-  | Int BigInt
+  | Int Int.Int
```

Comment:
Why not use a native type here?

Reply:
It's just like in CSL.

Reply:
I just wonder, since we're converting the entire thing to native types why leave such things as ints in their foreign CSL versions.

Reply:
My understanding is that

- it's hard to recreate it in pure PS (although possible - can be modeled as `(Sign, BigNum)`)
- we don't need to perform arithmetic operations on it for now

Same goes for `BigNum`.

CC @ngua what do you think?

Reply:
I'm fine with using the foreign `Int`